### PR TITLE
V2 Wizard: Add custom file system to request customizations (HMS-3690)

### PR DESCRIPTION
**Pull request by lucasgarfield**
The blueprint request now contains a field for the filesystem in its customizations.

### CODE REVIEW

File: src/Components/CreateImageWizardV2/steps/FileSystem/FileSystemConfiguration.tsx [hunk 1]
```diff
@@ -322,7 +322,7 @@ const MountpointSuffix = ({ partition }: MountpointSuffixPropTypes) => {
           })
         );
       }}
-      aria-label="text input example"
+      aria-label="mountpoint suffix"
```
Comment (mgold1234) [inline src/Components/CreateImageWizardV2/steps/FileSystem/FileSystemConfiguration.tsx:325]: can you please change it to "Mount point suffix text input" as it was in V1wizard?

File: src/Components/CreateImageWizardV2/utilities/requestMapper.tsx [hunk 2]
```diff
@@ -8,6 +8,7 @@ import {
   CreateBlueprintRequest,
   Customizations,
   DistributionProfileItem,
+  Filesystem,
```
Comment (mgold1234) [inline src/Components/CreateImageWizardV2/utilities/requestMapper.tsx:11]:
```suggestion
  FileSystem,
```

File: src/Components/CreateImageWizardV2/utilities/requestMapper.tsx [hunk 3]
```diff
@@ -317,6 +320,18 @@ const getOpenscapProfile = (state: RootState): OpenScap | undefined => {
   return undefined;
 };
 
+const getFileSystem = (state: RootState): Filesystem[] | undefined => {
+  const mode = selectFileSystemPartitionMode(state);
+  if (mode === 'manual') {
+    const partitions = selectPartitions(state);
+    const filesystem = partitions.map((partition) => {
```
Comment (mgold1234) [inline src/Components/CreateImageWizardV2/utilities/requestMapper.tsx:327]:
```suggestion
    const fileSystem = partitions.map((partition) => {
```

**Comment by mgold1234** [comment]
there is a problem with getFileSystem function at 'requestMapper.tsx when I create an image I got an error - 
```Blueprint could not be created
Status code 400: cannot parse request body: json: cannot unmarshal string into Go struct field Filesystem.customizations.filesystem.min_size of type uint64```

**Comment by lucasgarfield** [comment]
> there is a problem with getFileSystem function at 'requestMapper.tsx when I create an image I got an error -
> 
> ```
> Status code 400: cannot parse request body: json: cannot unmarshal string into Go struct field Filesystem.customizations.filesystem.min_size of type uint64```
> ```

Oh nice catch. The image builder API specifies the type of the min_size as `any` so I thought it would be okay to use a `string`. I'll change to `number`.

File: src/Components/CreateImageWizardV2/utilities/requestMapper.tsx (hunk 2 above)
Comment (lucasgarfield) [reply src/Components/CreateImageWizardV2/utilities/requestMapper.tsx:11]: I agree we should use `FileSystem` camel case but this is an import from the `image-builder` API so I'll suggest we keep it and don't rename it.

**Review by mgold1234 — APPROVED** [review]
/lgtm

**Pull request merged**